### PR TITLE
feat: [IOPLT-13] Increment Android versionCode on new version bump

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR improves the versionCode for android gradle file while bumping the version of the app during release process

### CODE REVIEW

File: scripts/changelog/gradle_updater.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable functional/immutable-data */
```
Comment (shadowsheep1) [inline scripts/changelog/gradle_updater.js:3]: This are great! Should we had a JIRA task to insert this lint exception to all other scripts?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: scripts/changelog/gradle_updater.js (hunk 1 above)
Comment (CrisTofani) [reply scripts/changelog/gradle_updater.js:3]: Let's refactor step by step on need what do you think?

File: android/app/build.gradle [hunk 2]
```diff
@@ -135,7 +135,7 @@ android {
         applicationId "it.pagopa.io.app"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode System.getenv('CIRCLE_BUILD_NUM') ? ("100" + System.getenv('CIRCLE_BUILD_NUM')).toInteger() : 1
+        versionCode 100154725
```
[the remark this answers is not in the document]
Comment (CrisTofani) [reply android/app/build.gradle:138]: yep this is the one on the `2.32.0-rc.2`

File: scripts/changelog/gradle_updater.js (hunk 1 above)
Comment (shadowsheep1) [reply scripts/changelog/gradle_updater.js:3]: I wouldn't do it in this PR, but I'd do another PR that address all other scripts in this section (folder `./scripts`). It's really annoying open a scripts and be flooded with many errors ;). What do you think?

Comment (CrisTofani) [reply scripts/changelog/gradle_updater.js:3]: Even if it is not that common to open these scripts, deal! 👍🏻

File: scripts/changelog/gradle_updater.js [hunk 3]
```diff
@@ -12,8 +15,9 @@
 const versionModule = require("./version_utility.js");
 
 const versionNameRegex = /(versionName ")(.+)(")/gm;
+const versionCodeRegex = /(versionCode )([0-9]+)/gm;
 
-module.exports.readVersion = function(contents) {
+module.exports.readVersion = function (contents) {
```
Comment (shadowsheep1) [inline scripts/changelog/gradle_updater.js:20]: Okay. According to [this](https://www.crockford.com/code.html), this is correct.

> if a function literal is anonymous, there should be one space between the word function and the

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**